editor/plugins/node_3d_editor_gizmos.cpp

```diff
@@ -3460,7 +3460,7 @@ String ReflectionProbeGizmoPlugin::get_handle_name(const EditorNode3DGizmo *p_gi
 
 Variant ReflectionProbeGizmoPlugin::get_handle_value(const EditorNode3DGizmo *p_gizmo, int p_id, bool p_secondary) const {
 	ReflectionProbe *probe = Object::cast_to<ReflectionProbe>(p_gizmo->get_node_3d());
-	return AABB(probe->get_extents(), probe->get_origin_offset());
+	return AABB(probe->get_size() / 2, probe->get_origin_offset());
```

Comment:
This isn't your fault, but the arguments are backwards here, they should be reversed as ``AABB()`` takes position then size. Also, since the label says "Size" now, this should return the size, not the extents
```suggestion
	return AABB(probe->get_origin_offset(), probe->get_size());
```

Reply:
Yeah I already figured, thanks! Should be fixed now.